src/warnet/cli/rpc.py

```diff
@@ -27,6 +27,8 @@ def rpc_call(rpc_method, params: dict[str, Any] | tuple[Any, ...] | None):
         logging.error(
             f"Error connecting to {url}. Is the server running and using the correct URL?"
         )
+        print("\nPlease consider running the following warnet command:")
+        print("  warcli cluster deploy\n")
```

Comment:
32c8f1ceab03a815693093483041529cc3f40291

why logging.error above but print() here? also maybe instead of hard coding a command that might change, just something like "is there a cluster deployed?"